src/types/posters.ts

```diff
@@ -1,6 +1,12 @@
+export interface Category {
+  category: string;
+  subcategory: string;
+}
+
 export type Poster = {
   authorName: string;
   categories: string[];
+  subcategories?: Array<Category>;
```

Comment:
Why does `Category` have both 'category' and 'subcategory' keys on it? And given it's only used in the `Poster`'s `subcategories` key.. should it not be a `Subcategory` at the very least?

Why is `Poster`'s `categories` of type `string[]`, yet `subcategories` is `Array<Category>`?

Why are we using `Array<foo>` instead of our standard `foo[]` syntax?